AudioKit/Common/Nodes/AKNode.swift

```diff
@@ -46,6 +46,7 @@ extension AVAudioConnectionPoint {
     }
     
     deinit {
+        //AKLog("* AKNode")
```

Comment:
Because AKNode is in a bunch of spots, I didn't leave that in - however in debugging the AmplitudeTracker deinit it was helpful to see when it was called. Technically, we don't need it in there! Could just remove it.